ansys/mapdl/core/mapdl.py

```diff
@@ -144,6 +146,78 @@ def __init__(self, loglevel='DEBUG', use_vtk=True, log_apdl=False,
 
         self._post = PostProcessing(self)
 
+    def query(self):
+        """Get instance of Query class containing inline functions of APDL.
+
+        Most of the results of these methods are shortcuts for specific
+        combinations of arguments supplied to :func:`ansys.mapdl.core.Mapdl.get`.
+
+        Currently implemented functions:
+
+        - ``centrx(e)`` - get the centroid x-coordinate of element `e`
+        - ``centry(e)`` - get the centroid y-coordinate of element `e`
+        - ``centrz(e)`` - get the centroid z-coordinate of element `e`
+        - ``nx(n)`` - get the x-coordinate of node `n`
+        - ``ny(n)`` - get the y-coordinate of node `n`
+        - ``nz(n)`` - get the z-coordinate of node `n`
+        - ``kx(k)`` - get the x-coordinate of keypoint `k`
+        - ``ky(k)`` - get the y-coordinate of keypoint `k`
+        - ``kz(k)`` - get the z-coordinate of keypoint `k`
+        - ``lx(n, lfrac)`` - X-coordinate of line ``n`` at length fraction ``lfrac``
+        - ``ly(n, lfrac)`` - Y-coordinate of line ``n`` at length fraction ``lfrac``
+        - ``lz(n, lfrac)`` - Z-coordinate of line ``n`` at length fraction ``lfrac``
+        - ``lsx(n, lfrac)`` - X-slope of line ``n`` at length fraction ``lfrac``
+        - ``lsy(n, lfrac)`` - Y-slope of line ``n`` at length fraction ``lfrac``
+        - ``lsz(n, lfrac)`` - Z-slope of line ``n`` at length fraction ``lfrac``
+        - ``ux(n)`` - get the structural displacement at node `n` in x
+        - ``uy(n)`` - get the structural displacement at node `n` in y
+        - ``uz(n)`` - get the structural displacement at node `n` in z
+        - ``rotx(n)`` - get the rotational displacement at node `n` in x
+        - ``roty(n)`` - get the rotational displacement at node `n` in y
+        - ``rotz(n)`` - get the rotational displacement at node `n` in z
+        - ``nsel(n)`` - get the selection status of node `n`
+        - ``ksel(k)`` - get the selection status of keypoint `k`
+        - ``lsel(n)`` - get the selection status of line `n`
+        - ``asel(a)`` - get the selection status of area `a`
+        - ``esel(n)`` - get the selection status of element `e`
+        - ``vsel(v)`` - get the selection status of volume `v`
+        - ``ndnext(n)`` - get the next selected node with a number greater than `n`.
+        - ``kpnext(k)`` - get the next selected keypoint with a number greater than `k`.
+        - ``lsnext(n)`` - get the next selected line with a number greater than `n`.
+        - ``arnext(a)`` - get the next selected area with a number greater than `a`.
+        - ``elnext(e)`` - get the next selected element with a number greater than `e`.
+        - ``vlnext(v)`` - get the next selected volume with a number greater than `v`.
+        - ``node(x, y, z)`` - get the node closest to coordinate (x, y, z)
+        - ``kp(x, y, z)`` - get the keypoint closest to coordinate (x, y, z)
```

Comment:
At some point when the dust settles from query, we should consider providing a reference to the API documentation rather than listing all the methods.  As this list grows (or should we choose to refactor), this will be hard to maintain.

Reply:
That's true! I've now updated the original issue with all the additional methods I've found since and it's over 50 now!
Plus in that list all the triple commands like KX KY and KZ have one entry but in this list they have 3.